spec/rubocop/cop/style/sample_spec.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 RSpec.describe RuboCop::Cop::Style::Sample, :config do
+  let(:ruby_version) { 2.7 }
```

Comment:
I'm a little worried that the tests won't run in Ruby 2.6 and Ruby 2.5. Can you tweak it as follows?

```diff
diff --git a/spec/rubocop/cop/style/sample_spec.rb b/spec/rubocop/cop/style/sample_spec.rb
index 4d5ef3742..e7675c3fd 100644
--- a/spec/rubocop/cop/style/sample_spec.rb
+++ b/spec/rubocop/cop/style/sample_spec.rb
@@ -1,8 +1,6 @@
 # frozen_string_literal: true

 RSpec.describe RuboCop::Cop::Style::Sample, :config do
-  let(:ruby_version) { 2.7 }
-
   shared_examples 'offense' do |wrong, right|
     it "registers an offense for #{wrong}" do
       expect_offense(<<~RUBY, wrong: wrong)
@@ -26,7 +24,9 @@ RSpec.describe RuboCop::Cop::Style::Sample, :config do
   it_behaves_like('offense', 'shuffle.last', 'sample')
   it_behaves_like('offense', 'shuffle[0]', 'sample')
   it_behaves_like('offense', 'shuffle[-1]', 'sample')
-  it_behaves_like('offense', 'shuffle[...3]', 'sample(3)')
+  context 'Ruby >= 2.7', :ruby27 do
+    it_behaves_like('offense', 'shuffle[...3]', 'sample(3)')
+  end
   it_behaves_like('offense', 'shuffle[0, 3]', 'sample(3)')
   it_behaves_like('offense', 'shuffle[0..3]', 'sample(4)')
   it_behaves_like('offense', 'shuffle[0...3]', 'sample(3)')
@@ -68,8 +68,10 @@ RSpec.describe RuboCop::Cop::Style::Sample, :config do
   it_behaves_like('accepts', 'shuffle[2..3]')           # empty if coll.size < 3
   it_behaves_like('accepts', 'shuffle[2..-3]')          # can't compute range size
   it_behaves_like('accepts', 'shuffle[foo..3]')         # can't compute range size
-  it_behaves_like('accepts', 'shuffle[3..]')            # can't compute range size
-  it_behaves_like('accepts', 'shuffle[3...]')           # can't compute range size
+  context 'Ruby >= 2.6', :ruby26 do
+    it_behaves_like('accepts', 'shuffle[3..]')            # can't compute range size
+    it_behaves_like('accepts', 'shuffle[3...]')           # can't compute range size
+  end
   it_behaves_like('accepts', 'shuffle[-4..-3]')         # nil if coll.size < 3
   it_behaves_like('accepts', 'shuffle[foo]')            # foo could be a Range
   it_behaves_like('accepts', 'shuffle[foo, 3]')         # nil if coll.size < foo
```

Reply:
Thanks for the tip, applied it so the tests run on Ruby 2.6 and below.